RFCs/rfc-0007-zkPassport.md

```diff
@@ -0,0 +1,106 @@
+
+For other chains, using Mina's zk state bridge, they could also leverage these proofs in their applications.
+
+Web2 applications may find use in this functionality as well. This seems particularly relevant as AI systems become increasingly powerful, and the risk this could pose to new and existing digital systems being overwhelmed by fake users.
+
+## Scenarios and Use Cases
+
+### Example Scenario 1: Proof-of-Uniqueness
+
+| Aspect           | Description |
+|------------------|-------------|
+| **Description**  | A zkApp wants a user to prove they are unique. |
+| **Requirements** | An implementation of zkPassport in the Attestation API Standard (see objectives), plus an on-chain nullifier that users can use with the zkApp. |
+| **Expected Outcome** | Via leveraging a simple library, users can make zkApps that are sybil-resistant. |
+| **Impact Analysis** | This would unlock many applications, such as one-person-one-vote, one-person-one account, Sybil-Resistant airdrops, etc. |
+
+### Example Scenario 2: Proof-of-nationality
+
+| Aspect           | Description |
+|------------------|-------------|
+| **Description**  | A zkApp wants a user to prove facts about what country they are / are not from. |
+| **Requirements** | An implementation of zkPassport in the Attestation API Standard (see objectives). |
+| **Expected Outcome** | Via leveraging a simple library, zkApps can require a user to prove facts about their nationality. |
+| **Impact Analysis** | This would be useful across web3, to make applications that are regulatory-compliant. |
+
+## Open Issues and Discussion Points
+
+A remaining question is exactly what new cryptography primitives would need to be implemented in o1js, if any new ones are needed. It looks like RSA, DSA, ECDSA, and Sha-xxx are used (see pages 15, 16, and 17 here[[4](https://www.icao.int/publications/Documents/9303_p12_cons_en.pdf)]).
+
+It seems countries keep databases of lost, stolen, and revoked travel documents[[5](https://www.icao.int/publications/Documents/9303_p2_cons_en.pdf)]. It doesn't seem like this is public however, so there may also have to be logic for invalidating one's old passport if one gets a new passport (due to the old one either being lost, stolen, revoked, or just expired).
+
+Lastly, a passport isn't something that everyone has or is able to get, so this is unlikely to work as a universal solution for proving personhood. That said, it could be a useful "anchor", for decentralized social-graph based approaches, which I think could be promising future work.
+
+This could be done via passport-holders verifying non-passport-holders. One way this could work is requiring multiple passport-holders, say 3, to attest to the validity of a non-passport-holder, and limiting the number of times they can do this (if 3 passport-holders are required, and each passport-holder gets 3 verifications, this would mean each passport-holder could verify up to one non-passport-holder on average).
```

Comment:
Social identity is really cool, I like this one! 💡

Reply:
This is interesting but really difficult to avoid collusion of passport holders to validate a fake/malicious non passport holder. The validated malicious non-holder can latter perform malicious acttions without any difficulty. The non-holder may not even exist and be just a way of the 3 holders to act together and perform some action. Having no traceability back from the initial holder approvers to the fake non-holder also makes it quite worse. 

Validating identity by social methods is much more complex. ProofOfHumanity has some way of doing it, we used anonymous trusted validators in the Identicon protocol (the idea that originated Socialcap), and there are others.

Reply:
BrightID is another viable identity model
-------------------------------------
Revoking identity and multiple validations could be another way to solve this

Reply:
Proof of uniqueness can be a solution here if user don't have passport or prefer not to share the data. Any kind of social system is always open to be gamed

Reply:
I dont think the solution is to track every single person all the time